packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CozyMarkerBuilder.java

```diff
@@ -9,25 +9,25 @@
 import android.graphics.Path;
 import android.graphics.Rect;
 import android.graphics.RectF;
+import android.util.LruCache;
 import android.graphics.Typeface;
 
 import androidx.core.content.res.ResourcesCompat;
 
-
 public class CozyMarkerBuilder {
     private final int shadowSize = 3;
     private final int priceMarkerTailSize;
     private final int padding;
     private final int size;
-    private final Bitmap blankClusterMarker;
+    private final MarkerCache markerCache;
     private final Typeface font;
 
     CozyMarkerBuilder(Context context) {
         size = getMarkerSize();
         padding = size / 3;
         priceMarkerTailSize = size / 6;
-        blankClusterMarker = getEmptyClusterBitmap(size);
```

Comment:
i've decided to remove this empty blank cluster marker because it will not be necessary considering we're storing the full markers in the cache.